projects/packages/sync/src/modules/class-callables.php

```diff
@@ -384,7 +384,7 @@ public function set_plugin_action_links() {
 			return;
 		}
 		$plugins = array_keys( Functions::get_plugins() );
```

Comment:
Without this it won't work right. 😉
```suggestion
		$plugins = Functions::get_plugins();
```